src/lms-harmonizer/edfi_lms_harmonizer/__main__.py

```diff
@@ -33,7 +33,17 @@ def _configure_logging(arguments: MainArguments) -> None:
 
 @catch_exceptions
 def _run(arguments: MainArguments) -> None:
-    print('base project')
+    global logger
+    engine: Engine = arguments.get_db_engine()
+
+    logger.info("Harmonizing Canvas LMS Users.")
+    engine.execute("EXEC lms.harmonize_lmsuser_canvas;")
+
+    logger.info("Harmonizing Google Classroom LMS Users.")
+    engine.execute("EXEC lms.harmonize_lmsuser_google_classroom;")
+
+    logger.info("Harmonizing Schoology LMS Users.")
+    engine.execute("EXEC lms.harmonize_lmsuser_schoology;")
```

Comment:
Must remember to test the exception handling. We do have the decorator at line 34 so I'm guessing that we'll have a good outcome.

Reply:
Yes, this behaved the way we want it to 😁